libp2p-networking/src/network/mod.rs

```diff
@@ -235,59 +235,6 @@ pub async fn gen_transport(
         .timeout(std::time::Duration::from_secs(20))
         .boxed())
 }
-// pub async fn development_transport(
```

Comment:
nit: Can we remove this

Reply:
Remove the commented code here? +1 that should absolutely be deleted. But I think I've deleted it?